src/Prod/README.md

```diff
@@ -0,0 +1,152 @@
+# Prod
+```haskell
+Prod Number
+```
+`Prod` is a `Monoid` that will combine two numbers using Multiplication.
```

Comment:
> `Prod` is a `Monoid` that will combine two numbers using Multiplication.

should probably use the phrasing:

`Prod` is a `Monoid` that will combine (2) `Number`s under multiplication.